x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/actions_log_empty_state.tsx

```diff
@@ -27,7 +27,7 @@ export const ActionsLogEmptyState = memo(
           title={
             <h2>
               {i18n.translate('xpack.securitySolution.actions_log.empty.title', {
```

Comment:
and this one to  `xpack.securitySolution.responseActionsHistory.empty.title`

Reply:
updated